core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -937,4 +938,56 @@ default HttpResponse recover(Function<? super Throwable, ? extends HttpResponse>
         requireNonNull(function, "function");
         return of(new RecoverableStreamMessage<>(this, function, /* allowResuming */ false));
     }
+
+    /**
+     * Recovers an {@link HttpResponse} exception to the specified {@link Throwable} class.
+     * when any error occurs before a {@link ResponseHeaders} is written.
```

Comment:
This isn't a complete sentence

```suggestion
     * Recovers a failed {@link HttpResponse} by switching to a returned fallback {@link HttpResponse}
     * when the thrown {@link Throwable} is the same type or a subtype of the
     * specified {@code causeClass}.
```

Reply:
Thanks! i will fix it